MAUI/Themes/Themes.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: post
+title: Applying themes for Syncfusion controls
+description: This section explains about theming in Syncfusion controls and how to override the key values for further customization
+platform: maui
+control: General
+documentation: ug
+---
+
+# Applying Themes in Maui Controls
+
+Syncfusion themes allow you apply colors across all the Syncfusion controls with a uniform approach and provide consistent look and feel to your applications. This section covers the following topics: 
```

Comment:
Syncfusion themes allow you to apply colors across all the Syncfusion controls with a uniform approach and provide a consistent look and feel to your applications. This section covers the following topics:

Reply:
Changes addressed.